dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -0,0 +1,45 @@
+class EvaluateRubricJob < ApplicationJob
+  queue_as :default
+
+  rescue_from(StandardError) do |exception|
+    if rack_env?(:development)
+      puts "EvaluateRubricJob Error: #{exception.full_message}"
+    end
+    raise
+  end
+
+  # 2D Map from unit name and level name, to the name of the lesson files in S3
+  # which will be used for AI evaluation.
+  # TODO: This is a temporary solution. After the pilot, we should at least make
+  # the S3 pointer editable on levelbuilder, and eventually make all of the data
+  # it points to editable there too.
+  UNIT_AND_LEVEL_TO_LESSON_S3_NAME = {
+    'csd3-2023' => {
+      'CSD U3 Interactive Card Final_2023' => 'CSD-2022-U3-L17',
+      'CSD U3 Sprites scene challenge_2023' => 'New-U3-2022-L10',
+      'CSD web project animated review_2023' => 'New-U3-2022-L13',
+      'CSD games sidescroll review_2023' => 'New-U3-2022-L20'
+    }
+  }
```

Comment:
You can see what files these point to here: https://s3.console.aws.amazon.com/s3/buckets/cdo-ai?region=us-east-1&prefix=teaching_assistant/lessons/CSD-2022-U3-L17/&showversions=false